docs/data/material/customization/palette/palette.md

```diff
@@ -210,6 +301,11 @@ const theme = createTheme({
 });
 ```
 
+:::warning
+The `contrastThreshold` parameter can produce counterproductive results.\
+Please verify that the [APCA](https://contrast.tools/?tab=apca) color contrast is improved (WCAG 3 [will use](https://typefully.com/u/DanHollick/t/sle13GMW2Brp) this new algorithm).
+:::
```

Comment:
I was trying to find a color for which using a contrast threshold of 4.5:1 would actually improve the APCA rating, and I couldn't. Do you think we should remove the 4.5:1 contrast threshold example above altogether?

Reply:
I think we can keep it like this


https://yatil.net/blog/wcag-3-is-not-ready-yet#update-2023-03-28

> So, should you use APCA?
The answer is two-fold:

> For private, non-commercial projects: Sure, go ahead and apply it to the best of your knowledge. There are no restrictions on what you can do, and it might even inform future developments in WCAG.

> For commercial and public projects: You are probably, through law or policy, required to conform to WCAG 2. That means you cannot rely on APCA for compliance (simply because it does not exist in WCAG 2).